src/promptflow/tests/test_configs/flows/simple_fetch_url/.gitattributes

```diff
@@ -0,0 +1 @@
+* text eol=lf
```

Comment:
So this will force all commit to use the same eol?

Reply:
yes, and the scope is only inside this folder

Reply:
why not making it for all flows, does this mean we need to add this to every flow that we use in a test?

Reply:
if we want this happen, we can simply move this file to `/src/promptflow/tests/test_configs/flows`, that's enough; but I think it's kind of a big change, and we don't really need that for this test: before this PR, line separator doesn't matter at all and nobody care it.

Reply:
and I will prefer to make whole repo LF, if we really depend on this later.

Reply:
Let's move it up a bit, as put it under flow directory will change flow snapshot

Reply:
offline synced with Clement, keep current status for now